pygate_grpc/client.py

```diff
@@ -1,5 +1,10 @@
-from pygate_grpc import health, faults, deals, ffs
+import sys
+import os
+
+sys.path.append(os.path.dirname(os.path.realpath(__file__)) + "/../")
```

Comment:
Just a minor sidenote :)) You can skip manually adding the module in PATH since pygate_grpc is already installed in pipenv's virtual environment. To activate it you can run `pipenv shell` and then run the examples normally and it should pick pygate_grpc automatically. Alternatively, you can run `pip install -e .` to install pygate_grpc locally in your environment.

Reply:
Ha, this is great. I didn't know before. Thanks for suggestion. This is fixed now.